source/grid/grid_tools.cc

```diff
@@ -556,7 +556,7 @@ namespace GridTools
     class ShiftPoint
```

Comment:
By definition a "vector" does not describe an action. what about just calling it "shift"?